typings/index.d.ts

```diff
@@ -303,13 +303,13 @@ export class Client extends BaseClient {
   public on<K extends keyof ClientEvents>(event: K, listener: (...args: ClientEvents[K]) => Awaited<void>): this;
   public on<S extends string | symbol>(
     event: Exclude<S, keyof ClientEvents>,
-    listener: (...args: unknown[]) => Awaited<void>,
+    listener: (...args: any[]) => Awaited<void>,
```

Comment:
This should remain as unknown, same with once and off

Reply:
Why?

Reply:
To force you to cast your data or check its type